README.md

```diff
@@ -9,14 +9,28 @@
  
 </div>
 
-_For the installation instructions, [click here](#installation)._
+## Contents:
 
-NNCF provides a suite of advanced algorithms for Neural Networks inference optimization in [OpenVINO&trade;](https://docs.openvino.ai/latest/home.html) with minimal accuracy drop.
+- [What is Neural Network Compression Framework (NNCF)?](#what-is-Neural-Network-Compression-Framework)
+- [Key features of NNCF](#key-features)
+- [Documentation](#documentation)
+- [Usage examples](#usage)
+- [Model compression tutorials and samples](#Model-compression-tutorials-and-samples)
+- [Third-party repository integration](#Third-party-repository-integration)
+- [System requirements](#System-requirements)
+- [Installation guide](#Installation-guide)
+- [Contributing guide](#Contributing-Guide)
+- [NNCF Compressed Model Zoo](#NNCF-Compressed-Model-Zoo)
+- [Citing](#Citing)
+- [Useful links](#Useful-links)
```

Comment:
These links aren't working, you're supposed to write references to header anchors in lowercase only.